docs/src/pages/components/geo/index.page.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Geo
+description: Amplify UI Geo provides UI components for maps and location search for popular front end frameworks.
+---
+
+import { Example } from '@/components/Example';
+import { Feature } from '@/components/Feature';
+import { Fragment } from '@/components/Fragment';
+import { XStateInspector } from '@/components/XStateInspector';
+import { Tabs, TabItem } from '@aws-amplify/ui-react';
+
+Amplify UI Geo provides UI components for maps and location search for popular front end frameworks, powered by
+[Amplify Geo APIs](https://docs.amplify.aws/lib/geo/getting-started/q/platform/js/) and
```

Comment:
just curious why do we need to break the lines?

Reply:
Guessing this is just prettier enforcing a line length.

Reply:
I think because it's just a really long line, so it helps with readability. I believe this doesn't affect the look and feel when it's displayed though.